### PR TITLE
Refactor using dex-js

### PR DESCRIPTION
**Pull request by anxolin**
Moves logic to `dex-js`.  For more info see: https://github.com/gnosis/dex-js/pull/28

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,8 +67,7 @@
     "ttypescript": "^1.5.8",
     "typechain": "^1.0.3",
     "typechain-target-web3-v1": "^1.0.0",
-    "typescript": "^3.7.2",
-    "typescript-transform-paths": "^1.1.13"
```
Comment (Velenir) [inline package.json]: Build breaks because you removed this plugin

Comment (anxolin) [reply package.json]: yep, saw that! I've removed that by mistake...
I've moved it to dex-js but it needs to be here too

File: src/helpers/index.ts [hunk 2]
```diff
@@ -1,3 +1,2 @@
-export * from './shutdown'
-export * from './error'
+// export * from '@gnosis.pm/dex-js'
```
Comment (alfetopito) [inline src/helpers/index.ts]: Dead code?

**Comment by anxolin** [comment]
> Should yarn.lock file be removed as well?

Good eye, it was a mistake to delete it

**Pull request merged**